scripts/verify-upstream.sh

```
@@ -0,0 +1,90 @@
+    echo "::error::Workflow SHA not provided and GITHUB_SHA not set"
+    exit 1
+fi
+
+echo "Verifying upstream has not changed from SHA: $WORKFLOW_SHA"
+
+# Force release branch to be at workflow sha
+echo "Resetting to workflow SHA..."
+git reset --hard "$WORKFLOW_SHA"
+
+# Determine upstream branch name
+UPSTREAM_BRANCH_NAME="$(git rev-parse --abbrev-ref @{upstream} 2>/dev/null || git rev-parse --abbrev-ref origin/$(git rev-parse --abbrev-ref HEAD) 2>/dev/null)"
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)